src/glue.c

```diff
@@ -475,16 +475,6 @@ void genObjFile(Module *m, bool multiobj)
         m->ssharedctor = callFuncsAndGates(m, &ssharedctors, (StaticDtorDeclarations *)&esharedctorgates, "__modsharedctor");
         m->sshareddtor = callFuncsAndGates(m, &sshareddtors, NULL, "__modshareddtor");
         m->stest = callFuncsAndGates(m, &stests, NULL, "__modtest");
-
-        if (m->doppelganger)
-            genModuleInfo(m);
-    }
-
-    if (m->doppelganger)
-    {
-        objc_Module_genmoduleinfo_classes();
```

Comment:
If I'm following the logic correctly, the only difference between the before and after for your patch (assuming betterC flag is not set) is that this call is now omitted. Is that right?